RNTester/js/RNTesterList.ios.js

```diff
@@ -272,6 +272,10 @@ const APIExamples: Array<RNTesterExample> = [
     supportsTVOS: false,
   },
   {
+    key: 'Dimensions',
+    module: require('./DimensionsExample'),
+  },
```

Comment:
Looks like you have to specify `supportsTVOS`, it causes a flow error.

Reply:
Whoops, did a blind copy from RNTesterList.android since the example works the exact same on both platform. Didn't notice the ios specific property.

Fixed.